### PR TITLE
Add docs for orders websocket channel

### PR DESCRIPTION
**Pull request by aan1092j**
Adding documentation for subscribing to, receiving updates from, and unsubscribing from the orders websocket channel.

### CODE REVIEW

File: docs/websocket.md [hunk 1]
```diff
@@ -307,3 +307,283 @@ eg:
    }
 }
 ```
+
```
Comment (djphillyg) [inline docs/websocket.md]: this documentation should be in the same section as the one above for `orderbook`, not separated by the `operations` documentation

File: docs/websocket.md [hunk 2]
```diff
@@ -307,3 +307,283 @@ eg:
...
+      }
+    ]
+  }
+}
+```
+
+#### Updates
+
+Updates to the orders are posted on the channel. If an order is filled, the corresponding fills will also be
+sent on the channel. eg:
```
Comment (AntonioJuliano) [inline docs/websocket.md]: Just mention that orders / fills are in the same schema as the V2 http API

Comment (aan1092j) [reply docs/websocket.md]: The v2 http schema for orders and fills is not on master yet, should I add it.

Comment (AntonioJuliano) [reply docs/websocket.md]: Phil's about to add it, so yeah we should

Comment (aan1092j) [reply docs/websocket.md]: added

File: docs/websocket.md (hunk 1 above)
Comment (aan1092j) [reply docs/websocket.md]: fixed

**Pull request merged**